packages/blockchain-wallet-v4/src/network/api/profile/index.ts

```diff
@@ -47,6 +55,38 @@ export default ({ authorizedGet, authorizedPost, authorizedPut, nabuUrl, post, r
     })
   }
 
+  const createExchangeUser = (countryCode, referrerUsername, retailToken, tuneTid) => {
+    return post({
+      contentType: 'application/json',
+      data: {
+        countryCode,
+        referrerUsername,
+        retailToken,
+        tuneTid
+      },
+      endPoint: '/mercury/users',
+      url: nabuUrl
+    })
+  }
+
+  const getExchangeAuthToken = (exchangeLifetimeToken, usersCredentialsId, retailToken) => {
+    return authorizedPost({
+      contentType: 'application/json',
+      data: {
+        retailToken,
+        usersCredentialsId
+      },
+      endPoint: '/mercury/auth',
+      headers: {
+        Authorization: `Bearer ${exchangeLifetimeToken}`,
+        'X-CLIENT-TYPE': 'WEB',
+        'X-DEVICE-ID': null,
+        'x-app-version': '6.11.1'
```

Comment:
double check what web should be sending here with Mr. P

Reply:
check rest of codebase for some issue(?)

Reply:
only in one other place, checked with Pavel and their mandatory